docs/modules/ROOT/pages/integrating-with-existing-deployments-collapsed.adoc

```diff
@@ -0,0 +1,252 @@
+= Integrating CMOS with Existing Monitoring Stacks
+:icons: font
+:imagesdir: ../assets/images
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+[abstract]
+You can integrate CMOS with your existing Prometheus, Alertmanager, and Grafana monitoring system. 
+This tutorial documents how to configure each component in order to achieve this.
+
+include::partial$tutorial.adoc[]
+
+== Pre-requisites
+
+The environment you wish to integrate CMOS into must have:
+
+. At least one **Couchbase Server** node;
+. **Prometheus**, with **Alertmanager** installed;
+. **Grafana**, configured to use the Prometheus server as a Data Source;
+
+==== Optional ====
+
+- **Docker**, if using the Cluster Monitor's Docker image instead of the binary.
+
+== Installation
+
+=== Install Cluster Monitor
+
+The Cluster Monitor can be installed as a binary (`.rpm`), or by downloading a Docker image.
+
+**Binary installation**
+
+_Binary installation steps.... .rpm just install using package manager?..._
+
+**Docker installation**
+
+[NOTE]
+====
+We plan for the image to be distributed on DockerHub, but currently this is not available. 
+====
+
+Run the Docker container, using the preview image given to you by Couchbase:
+[source, console]
+----
+docker run -d --rm \
+  -p 7196:7196 \ <.>
+  --network cmos \
+  --prometheus-label-selector job=couchbase-server <.>
+  --couchbase-user Administrator
+  --couchbase-password password
+  --name cluster_monitor \
+  <YOUR IMAGE HERE>
+----
+
+ALSO ADD ALERTMANAGER URLS to configure cluster monitor to send to it
+
+<.> If you are using TLS, you will need to add `-p 7197:7197` to also expose port `7197`.
+<.> `job=couchbase-server` is a label we will create later in this tutorial when configuring Prometheus' `scrape_config` to scrape our Couchbase Server nodes.
+
+=== Configure Cluster Monitor
+
+Navigate to the Web GUI, which listens by default on:
+- HTTP: 7196
+- HTTPS: 7197
```

Comment:
We're not exposing 7197 in the `docker run` above - let's either change that to `7196-7197:7196-7197` or remove it here

Reply:
> If you are using TLS, you will need to add `-p 7197:7197` to also expose port `7197`.

Reply:
Derp